mobile/src/main/java/org/openhab/habdroid/ui/preference/UrlInputPreference.kt

```diff
@@ -116,8 +130,8 @@
         companion object {
             private const val KEY_TITLE = "title"
 
-            fun newInstance(key: String, title: CharSequence): PrefFragment {
-                val f = PrefFragment()
+            fun newInstance(key: String, title: CharSequence, isHttpEnabled: Boolean): PrefFragment {
+                val f = PrefFragment(isHttpEnabled)
```

Comment:
Fragments must not have constructor parameters. Just put the boolean into the arguments bundle as well.